IFC4x3/Properties/i/InputCurrent_3PQ23h6ZXEi8EVCBxbM4g3/Documentation.md

```diff
@@ -0,0 +1 @@
+The actual current and operable range as input of the equipment.
```

Comment:
Is it a range?  Should it be a bounded property?

Reply:
Agree, bounded value. commit : 0bb5ae0d772bc5a99c0bea0886d3f8ed41fd2f5d

Reply:
Using RatedCurrent from Pset_ElectricDeviceCommon. Commit : f953d0e8936657c3a245b91271c0ba9406c565ef

Reply:
Don't see this change in the commit.